config.json.sample

```diff
@@ -20,12 +19,12 @@
     	"S3" : {
         }, 
 
-        "OneClick" : {
+        "ThirdPartyProvider" : {
```

Comment:
I don't think it's very useful to have a sample OneClick configuration (of all the things you might need a sample configuration for) in `config.json.sample`. But it makes sense to have examples, so I don't object very strongly to it going in. 

I do object to calling it "ThirdPartyProvider" instead of "OneClick". If you call it "OneClick" then it can be copied and modified by someone who wants to configure OneClick. If it's called "ThirdPartyProvider" I don't see it enlightening anyone.